src/ert/_c_wrappers/enkf/enkf_obs.py

```diff
@@ -36,18 +42,57 @@
 
 DEFAULT_TIME_DELTA = timedelta(seconds=30)
 
+USE_NEW_PARSER_BY_DEFAULT = True
```

Comment:
Use unique name for this? This is the same as the original lark implementation.

Reply:
Fixed